src/flag_gems/ops/gelu.py

```diff
@@ -31,9 +32,56 @@ def gelu_tanh(x):
     return output
 
 
+@pointwise_dynamic(promotion_methods=[(0, 1, "DEFAULT")])
+@triton.jit
+def gelu_backward_none(x, dy):
+    scale1: tl.constexpr = 0.7071067811
```

Comment:
Add some comment on the value of those constants.

```python
scale1: tl.constexpr = 1 / math.sqrt(2)
scale2: tl.constexpr = 1 / math.sqrt(2 * math.pi)
``` 

to help understanding the formula.